disnake/sku.py

```diff
@@ -81,3 +86,69 @@ def created_at(self) -> datetime.datetime:
     def flags(self) -> SKUFlags:
         """:class:`SKUFlags`: Returns the SKU's flags."""
         return SKUFlags._from_value(self._flags)
+
+    async def subscriptions(
+        self,
+        *,
+        limit: Optional[int] = 50,
+        before: Optional[SnowflakeTime] = None,
+        after: Optional[SnowflakeTime] = None,
+        user: Optional[Snowflake] = None,
```

Comment:
I haven't tested it, but assuming the documentation is accurate - it says this is "Required except for OAuth queries", so this should be a required parameter here (it's fine if the iterator itself still supports `Optional[User]`, since it's internal)